.github/workflows/ci.yaml

```diff
@@ -0,0 +1,47 @@
+name: Ci
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+  pull_request:
+    branches:
+      - master
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  CACHE_KEY: 0
+
+jobs:
+  all:
+    name: All
+    runs-on: ubuntu-latest
```

Comment:
I considered adding CI for different platforms, since github actions supports that. This PR is meant to replace the defunct travis-based CI (which only supported linux), and as I wanted to keep this PR small, I didn't want to introduce that here.